ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/admin/console/components/editDialog/editDialog.html

```diff
@@ -29,9 +29,24 @@
                                value="${model.packageModel.version}"
                                is="coral-textfield">
                     </div>
-                    <div class="coral-Form-fieldwrapper">
+                    <div class="coral-Well">
```

Comment:
Please use usual Switch with 'SQL2 Query Package' text

Reply:
Done. Switch text was renamed to 'SQL2 Query Package'